index.html

```diff
@@ -1754,14 +1759,14 @@ <h5>
           </li>
           <li>
             <p data-lang="en"><span class="leadin">Horizontal ellipsis.</span> </p>
-            <p data-lang="zh-hans" class="checkme"><span class="leadin">删节号／省略号</span></p>
+            <p data-lang="zh-hans"><span class="leadin">省略号／删节号</span></p>
             <p data-lang="zh-hant"><span class="leadin">刪節號／省略号</span></p>
-            <p data-lang="en">Use two <span class="uname">U+2026 HORIZONTAL ELLIPSIS</span> […] to takes one character height and two character widths: [……] .</p>
-            <p data-lang="zh-hans" class="checkme">删节号／省略号使用连续二个<span class="uname" translate="no">U+2026 HORIZONTAL ELLIPSIS</span> […]，占两个汉字空间、包含六个居中省略点。[……]</p>
-            <p data-lang="zh-hant">刪節號／省略號使用連續二個<span class="uname">U+2026 HORIZONTAL ELLIPSIS</span> […]，佔二個漢字空間、包含居中六個省略點。[……]</p>
+            <p data-lang="en">Use two <span class="uname">U+2026 HORIZONTAL ELLIPSIS</span> […] to takes one character height and two character widths: [……].</p>
```

Comment:
the example here, and in the previous subsection after "Long dashes" need to have language tags so that they are rendered correctly (using a chinese font).

Also, the syntax of this sentence is a little odd. How about:

"When two U+2026 HORIZONTAL ELLIPSIS characters are used together for ellipsis, they are each the height of one character and the width of two characters."

Or, following a substitution you made earlier:

"When two U+2026 HORIZONTAL ELLIPSIS characters are used together for ellipsis, they are, together, the height of one Hanzi character and the width of two Hanzi characters."

Reply:
"Hanzi character" seems odd to me, because "zi" means "character", and thus "Hanzi character" sounds like "Han character character". What about "Chinese character"?

Reply:
I don't remember the conclusion, but there was a thread about this in the early days of the project at https://github.com/w3c/clreq/issues/11  

At couple of additional notes, fwiw:
1. 'kanji character' is often used in English to refer to han characters used in Japanese. 
2. most English speakers won't be aware of the repetition in 'Hanzi character'
3. 2 hanzis definitely sounds wierd to me, whereas 2 hanzi/chinese/han characters sounds ok

hth

Reply:
@r12a Thanks and done!

@xfq I forgot the conclusion too (LoL). I’ll use Hanzi Character here. We can discuss this again in the next meeting and let Richard know about our decision.

Reply:
fwiw, i think the most english way to refer to these characters is as 'Han characters'. As i mentioned before, however, i don't think english speaking people would worry about 'hanzi character' any more than they worry about 'kanji character' (which is actually the _normal_ way to refer to han characters used in Japanese (as well as to the native Japanese ones that look like han characters)).